rest_framework/fields.py

```diff
@@ -1432,28 +1432,44 @@ class ChoiceField(Field):
     html_cutoff = None
     html_cutoff_text = _('More than {count} items...')
 
-    def __init__(self, choices, **kwargs):
+    def __init__(self, choices, underlying_field=None, **kwargs):
```

Comment:
Not so good thing with passing a `Field` instance is that you may end up passing same arguments twice (to `ChoiceField` and to the underlying field). 

Alternatively we can pass `underlying_field_class` and `underlying_field_additional_kwargs` to tell `ChoiceField` how to create it. `ChoiceField` will add some kwargs to the underlying field by itself. 

Or maybe we can modify `underlying_field` after creation (can we?)